scanner/indexer/indexer.go

```diff
@@ -67,6 +68,10 @@ var (
 	// remoteTransport is the http.RoundTripper to use when talking to image registries.
 	remoteTransport         = proxiedRemoteTransport(false)
 	insecureRemoteTransport = proxiedRemoteTransport(true)
+
+	// regsNoRange is a set of registries which do not accept the Range HTTP header.
+	// This is used for logging purposes, only.
+	regsNoRange sync.Map
```

Comment:
In case anyone is curious why I chose `sync.Map` instead of a regular `map[string]struct{}`:

https://pkg.go.dev/sync#Map
```
The Map type is optimized for two common use cases: (1) when the entry for a given key is only ever written once but read many times, as in caches that only grow, or (2) when multiple goroutines read, write, and overwrite entries for disjoint sets of keys. In these two cases, use of a Map may significantly reduce lock contention compared to a Go map paired with a separate Mutex or RWMutex.
```